ts/features/fims/screens/FimsFlowHandlerScreen.tsx

```diff
@@ -77,7 +71,7 @@ export const FimsFlowHandlerScreen = (
     consentsPot,
     pot.toOption,
     O.fold(
-      () => <FimsErrorBody title="generic error" />,
+      () => <FimsErrorBody title={I18n.t("global.genericError")} />,
```

Comment:
If FIMSErrorBody is a component, let's put it on its own file (including its properties), otherwise let's transform it to a function

Reply:
As `resolved` above, let's leave it as it is